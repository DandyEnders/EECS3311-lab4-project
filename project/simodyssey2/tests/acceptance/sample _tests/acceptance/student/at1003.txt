move(W)
move(NE)
wormhole
test(58,84,84,84,84)
move(S)
move(W)
move(N)
move(NE)
abort
move(S)
abort
move(E)
move(SE)
liftoff
move(W)
move(SW)
move(W)
move(SW)
land
move(W)
move(N)
wormhole
move(NW)
move(SW)
move(W)
move(S)
move(W)
