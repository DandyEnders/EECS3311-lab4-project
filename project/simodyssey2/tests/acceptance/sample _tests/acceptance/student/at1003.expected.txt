  state:0.0, ok
  Welcome! Try test(3,5,7,15,30)
->move(W)
  state:0.1, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.2, error
  Negative on that request:no mission in progress.
->wormhole
  state:0.3, error
  Negative on that request:no mission in progress.
->test(58,84,84,84,84)
  state:1.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],[1,A],-,-
    [1,2]->[2,A],[3,A],[-8,Y],-
    [1,3]->[4,A],[5,A],-,-
    [1,4]->[6,A],-,-,-
    [1,5]->[7,J],[-11,*],-,-
    [2,1]->[8,J],[9,A],-,-
    [2,2]->[10,A],[-6,Y],-,-
    [2,3]->[11,A],-,-,-
    [2,4]->-,-,-,-
    [2,5]->[12,J],[13,A],[14,J],[-4,*]
    [3,1]->[15,A],[16,A],-,-
    [3,2]->[17,J],[18,A],-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[19,A],[20,J],[21,A],[-3,W]
    [3,5]->[22,J],[23,A],[24,A],[-7,W]
    [4,1]->[25,A],-,-,-
    [4,2]->-,-,-,-
    [4,3]->[26,J],-,-,-
    [4,4]->[27,A],[28,A],[-10,W],-
    [4,5]->[29,A],-,-,-
    [5,1]->[30,A],[31,A],-,-
    [5,2]->[32,A],[-5,Y],-,-
    [5,3]->[33,A],[34,A],[-2,W],-
    [5,4]->[35,A],[-9,Y],-,-
    [5,5]->[36,A],[37,J],-,-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,W]->
    [-9,Y]->Luminosity:2
    [-8,Y]->Luminosity:2
    [-7,W]->
    [-6,Y]->Luminosity:2
    [-5,Y]->Luminosity:2
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,A]->turns_left:1
    [2,A]->turns_left:0
    [3,A]->turns_left:1
    [4,A]->turns_left:0
    [5,A]->turns_left:0
    [6,A]->turns_left:0
    [7,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [8,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [9,A]->turns_left:2
    [10,A]->turns_left:2
    [11,A]->turns_left:1
    [12,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [13,A]->turns_left:2
    [14,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [15,A]->turns_left:2
    [16,A]->turns_left:2
    [17,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [18,A]->turns_left:0
    [19,A]->turns_left:1
    [20,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [21,A]->turns_left:0
    [22,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [23,A]->turns_left:1
    [24,A]->turns_left:1
    [25,A]->turns_left:0
    [26,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [27,A]->turns_left:0
    [28,A]->turns_left:0
    [29,A]->turns_left:2
    [30,A]->turns_left:1
    [31,A]->turns_left:1
    [32,A]->turns_left:2
    [33,A]->turns_left:2
    [34,A]->turns_left:2
    [35,A]->turns_left:0
    [36,A]->turns_left:2
    [37,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EA--   AAY-   AA--   A---   J*--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    JA--   AY--   A---   ----   JAJ*   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    AA--   JA--   O---   AJAW   JAAW   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    A---   ----   J---   AAW-   A---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    AA--   AY--   AAW-   AY--   AJ--   
->move(S)
  state:2.0, mode:test, ok
  Movement:
    [0,E]:[1,1,1]->[2,1,3]
    [2,A]:[1,2,1]->[5,1,3]
    [4,A]:[1,3,1]->[5,3,4]
    [5,A]:[1,3,2]->[5,2,3]
    [6,A]:[1,4,1]->[5,4,3]
    [7,J]:[1,5,1]->[5,4,4]
      destroyed [6,A] at [5,4,3]
      destroyed [35,A] at [5,4,1]
    [18,A]:[3,2,2]->[3,3,2]
    [20,J]:[3,4,2]->[3,3,2]
    [21,A]:[3,4,3]
    [25,A]:[4,1,1]->[4,2,1]
    [27,A]:[4,4,1]->[5,4,1]
      destroyed [7,J] at [5,4,4]
    [28,A]:[4,4,2]->[4,3,2]
      destroyed [26,J] at [4,3,1]
  Sectors:
    [1,1]->-,[1,A],-,-
    [1,2]->-,[3,A],[-8,Y],-
    [1,3]->-,-,-,-
    [1,4]->-,-,-,-
    [1,5]->-,[-11,*],-,-
    [2,1]->[8,J],[9,A],[0,E],-
    [2,2]->[10,A],[-6,Y],-,-
    [2,3]->[11,A],-,-,-
    [2,4]->-,-,-,-
    [2,5]->[12,J],[13,A],[14,J],[-4,*]
    [3,1]->[15,A],[16,A],-,-
    [3,2]->[17,J],-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[19,A],-,[21,A],[-3,W]
    [3,5]->[22,J],[23,A],[24,A],[-7,W]
    [4,1]->-,-,-,-
    [4,2]->[25,A],-,-,-
    [4,3]->-,[28,A],-,-
    [4,4]->-,-,[-10,W],-
    [4,5]->[29,A],-,-,-
    [5,1]->[30,A],[31,A],[2,A],-
    [5,2]->[32,A],[-5,Y],[5,A],-
    [5,3]->[33,A],[34,A],[-2,W],[4,A]
    [5,4]->[27,A],[-9,Y],-,-
    [5,5]->[36,A],[37,J],-,-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,W]->
    [-9,Y]->Luminosity:2
    [-8,Y]->Luminosity:2
    [-7,W]->
    [-6,Y]->Luminosity:2
    [-5,Y]->Luminosity:2
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:2/3, life:3/3, landed?:F
    [1,A]->turns_left:0
    [2,A]->turns_left:2
    [3,A]->turns_left:0
    [4,A]->turns_left:2
    [5,A]->turns_left:0
    [8,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [9,A]->turns_left:1
    [10,A]->turns_left:1
    [11,A]->turns_left:0
    [12,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [13,A]->turns_left:1
    [14,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [15,A]->turns_left:1
    [16,A]->turns_left:1
    [17,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [19,A]->turns_left:0
    [21,A]->turns_left:2
    [22,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [23,A]->turns_left:0
    [24,A]->turns_left:0
    [25,A]->turns_left:2
    [27,A]->turns_left:1
    [28,A]->turns_left:2
    [29,A]->turns_left:1
    [30,A]->turns_left:0
    [31,A]->turns_left:0
    [32,A]->turns_left:1
    [33,A]->turns_left:1
    [34,A]->turns_left:1
    [36,A]->turns_left:1
    [37,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
  Deaths This Turn:
    [6,A]->turns_left:N/A,
      Asteroid got imploded by janitaur (id: 7) at Sector:5:4
    [35,A]->turns_left:N/A,
      Asteroid got imploded by janitaur (id: 7) at Sector:5:4
    [18,A]->turns_left:N/A,
      Asteroid got devoured by blackhole (id: -1) at Sector:3:3
    [20,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:N/A,
      Janitaur got devoured by blackhole (id: -1) at Sector:3:3
    [7,J]->fuel:5/5, load:2/2, actions_left_until_reproduction:1/2, turns_left:N/A,
      Janitaur got destroyed by asteroid (id: 27) at Sector:5:4
    [26,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:N/A,
      Janitaur got destroyed by asteroid (id: 28) at Sector:4:3
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -A--   -AY-   ----   ----   -*--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    JAE-   AY--   A---   ----   JAJ*   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    AA--   J---   O---   A-AW   JAAW   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   A---   -A--   --W-   A---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    AAA-   AYA-   AAWA   AY--   AJ--   
->move(W)
  state:2.1, mode:test, error
  Cannot transfer to new location as it is full.
->move(N)
  state:3.0, mode:test, ok
  Movement:
    [0,E]:[2,1,3]->[1,1,1]
    [1,A]:[1,1,2]->[2,2,3]
    [3,A]:[1,2,2]->[5,1,4]
    [5,A]:[5,2,3]->[4,3,1]
    [8,J]:[2,1,1]
      destroyed [9,A] at [2,1,2]
    [11,A]:[2,3,1]->[1,4,1]
    [12,J]:[2,5,1]->[1,5,1]
    [14,J]:[2,5,3]->[3,4,2]
      destroyed [19,A] at [3,4,1]
      destroyed [21,A] at [3,4,3]
    [17,J]:[3,2,1]->[4,1,1]
    [22,J]:[3,5,1]->[4,1,2]
    [23,A]:[3,5,2]->[3,1,3]
    [24,A]:[3,5,3]->[4,4,1]
    [30,A]:[5,1,1]->[5,5,3]
      destroyed [37,J] at [5,5,2]
    [31,A]:[5,1,2]->[5,5,2]
  Sectors:
    [1,1]->[0,E],-,-,-
    [1,2]->-,-,[-8,Y],-
    [1,3]->-,-,-,-
    [1,4]->[11,A],-,-,-
    [1,5]->[12,J],[-11,*],-,-
    [2,1]->[8,J],-,-,-
    [2,2]->[10,A],[-6,Y],[1,A],-
    [2,3]->-,-,-,-
    [2,4]->-,-,-,-
    [2,5]->-,[13,A],-,[-4,*]
    [3,1]->[15,A],[16,A],[23,A],-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,[14,J],-,[-3,W]
    [3,5]->-,-,-,[-7,W]
    [4,1]->[17,J],[22,J],-,-
    [4,2]->[25,A],-,-,-
    [4,3]->[5,A],[28,A],-,-
    [4,4]->[24,A],-,[-10,W],-
    [4,5]->[29,A],-,-,-
    [5,1]->-,-,[2,A],[3,A]
    [5,2]->[32,A],[-5,Y],-,-
    [5,3]->[33,A],[34,A],[-2,W],[4,A]
    [5,4]->[27,A],[-9,Y],-,-
    [5,5]->[36,A],[31,A],[30,A],-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,W]->
    [-9,Y]->Luminosity:2
    [-8,Y]->Luminosity:2
    [-7,W]->
    [-6,Y]->Luminosity:2
    [-5,Y]->Luminosity:2
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:1/3, life:3/3, landed?:F
    [1,A]->turns_left:1
    [2,A]->turns_left:1
    [3,A]->turns_left:0
    [4,A]->turns_left:1
    [5,A]->turns_left:2
    [8,J]->fuel:5/5, load:1/2, actions_left_until_reproduction:1/2, turns_left:2
    [10,A]->turns_left:0
    [11,A]->turns_left:0
    [12,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [13,A]->turns_left:0
    [14,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [15,A]->turns_left:0
    [16,A]->turns_left:0
    [17,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [22,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:1
    [23,A]->turns_left:2
    [24,A]->turns_left:1
    [25,A]->turns_left:1
    [27,A]->turns_left:0
    [28,A]->turns_left:1
    [29,A]->turns_left:0
    [30,A]->turns_left:0
    [31,A]->turns_left:2
    [32,A]->turns_left:0
    [33,A]->turns_left:0
    [34,A]->turns_left:0
    [36,A]->turns_left:0
  Deaths This Turn:
    [9,A]->turns_left:N/A,
      Asteroid got imploded by janitaur (id: 8) at Sector:2:1
    [19,A]->turns_left:N/A,
      Asteroid got imploded by janitaur (id: 14) at Sector:3:4
    [21,A]->turns_left:N/A,
      Asteroid got imploded by janitaur (id: 14) at Sector:3:4
    [37,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:N/A,
      Janitaur got destroyed by asteroid (id: 30) at Sector:5:5
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   --Y-   ----   A---   J*--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    J---   AYA-   ----   ----   -A-*   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    AAA-   ----   O---   -J-W   ---W   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    JJ--   A---   AA--   A-W-   A---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    --AA   AY--   AAWA   AY--   AAA-   
->move(NE)
  state:4.0, mode:test, ok
  Movement:
    [0,E]:[1,1,1]->[5,2,3]
    [3,A]:[5,1,4]->[4,5,2]
    [10,A]:[2,2,1]->[3,3,2]
    [11,A]:[1,4,1]->[2,4,1]
    [13,A]:[2,5,2]->[1,5,3]
      destroyed [12,J] at [1,5,1]
    [15,A]:[3,1,1]->[2,5,1]
    [16,A]:[3,1,2]->[4,1,3]
      destroyed [17,J] at [4,1,1]
      destroyed [22,J] at [4,1,2]
    [27,A]:[5,4,1]->[4,4,2]
    [29,A]:[4,5,1]->[4,4,4]
    [30,A]:[5,5,3]
    [32,A]:[5,2,1]->[4,3,3]
    [33,A]:[5,3,1]->[1,4,1]
    [34,A]:[5,3,2]->[4,3,4]
    [36,A]:[5,5,1]->[5,4,1]
  Sectors:
    [1,1]->-,-,-,-
    [1,2]->-,-,[-8,Y],-
    [1,3]->-,-,-,-
    [1,4]->[33,A],-,-,-
    [1,5]->-,[-11,*],[13,A],-
    [2,1]->[8,J],-,-,-
    [2,2]->-,[-6,Y],[1,A],-
    [2,3]->-,-,-,-
    [2,4]->[11,A],-,-,-
    [2,5]->[15,A],-,-,[-4,*]
    [3,1]->-,-,[23,A],-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,[14,J],-,[-3,W]
    [3,5]->-,-,-,[-7,W]
    [4,1]->-,-,[16,A],-
    [4,2]->[25,A],-,-,-
    [4,3]->[5,A],[28,A],[32,A],[34,A]
    [4,4]->[24,A],[27,A],[-10,W],[29,A]
    [4,5]->-,[3,A],-,-
    [5,1]->-,-,[2,A],-
    [5,2]->-,[-5,Y],[0,E],-
    [5,3]->-,-,[-2,W],[4,A]
    [5,4]->[36,A],[-9,Y],-,-
    [5,5]->-,[31,A],[30,A],-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,W]->
    [-9,Y]->Luminosity:2
    [-8,Y]->Luminosity:2
    [-7,W]->
    [-6,Y]->Luminosity:2
    [-5,Y]->Luminosity:2
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:2/3, life:3/3, landed?:F
    [1,A]->turns_left:0
    [2,A]->turns_left:0
    [3,A]->turns_left:0
    [4,A]->turns_left:0
    [5,A]->turns_left:1
    [8,J]->fuel:5/5, load:1/2, actions_left_until_reproduction:1/2, turns_left:1
    [11,A]->turns_left:1
    [13,A]->turns_left:1
    [14,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:0
    [15,A]->turns_left:2
    [16,A]->turns_left:2
    [23,A]->turns_left:1
    [24,A]->turns_left:0
    [25,A]->turns_left:0
    [27,A]->turns_left:0
    [28,A]->turns_left:0
    [29,A]->turns_left:1
    [30,A]->turns_left:0
    [31,A]->turns_left:1
    [32,A]->turns_left:0
    [33,A]->turns_left:1
    [34,A]->turns_left:1
    [36,A]->turns_left:1
  Deaths This Turn:
    [10,A]->turns_left:N/A,
      Asteroid got devoured by blackhole (id: -1) at Sector:3:3
    [12,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:N/A,
      Janitaur got destroyed by asteroid (id: 13) at Sector:1:5
    [17,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:N/A,
      Janitaur got destroyed by asteroid (id: 16) at Sector:4:1
    [22,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:N/A,
      Janitaur got destroyed by asteroid (id: 16) at Sector:4:1
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   --Y-   ----   A---   -*A-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    J---   -YA-   ----   A---   A--*   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    --A-   ----   O---   -J-W   ---W   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    --A-   A---   AAAA   AAWA   -A--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    --A-   -YE-   --WA   AY--   -AA-   
->abort
  state:4.1, ok
  Mission aborted. Try test(3,5,7,15,30)
->move(S)
  state:4.2, error
  Negative on that request:no mission in progress.
->abort
  state:4.3, error
  Negative on that request:no mission in progress.
->move(E)
  state:4.4, error
  Negative on that request:no mission in progress.
->move(SE)
  state:4.5, error
  Negative on that request:no mission in progress.
->liftoff
  state:4.6, error
  Negative on that request:no mission in progress.
->move(W)
  state:4.7, error
  Negative on that request:no mission in progress.
->move(SW)
  state:4.8, error
  Negative on that request:no mission in progress.
->move(W)
  state:4.9, error
  Negative on that request:no mission in progress.
->move(SW)
  state:4.10, error
  Negative on that request:no mission in progress.
->land
  state:4.11, error
  Negative on that request:no mission in progress.
->move(W)
  state:4.12, error
  Negative on that request:no mission in progress.
->move(N)
  state:4.13, error
  Negative on that request:no mission in progress.
->wormhole
  state:4.14, error
  Negative on that request:no mission in progress.
->move(NW)
  state:4.15, error
  Negative on that request:no mission in progress.
->move(SW)
  state:4.16, error
  Negative on that request:no mission in progress.
->move(W)
  state:4.17, error
  Negative on that request:no mission in progress.
->move(S)
  state:4.18, error
  Negative on that request:no mission in progress.
->move(W)
  state:4.19, error
  Negative on that request:no mission in progress.
